openfisca-web-api/README.md

```diff
@@ -53,3 +30,10 @@ Computations done by <a href="https://openfisca.fr">OpenFisca</a>, the <a href="
 ```html
 Calculs effectués par <a href="https://openfisca.fr">OpenFisca</a>, le moteur <a href="https://choosealicense.com/licenses/agpl-3.0/" title="AGPL-3.0">libre et ouvert</a> du système social et fiscal. Code source disponible sur <a href="https://github.com/openfisca">github.com/openfisca</a>.
 ```
+## Hosting an API instance
+
+To allow app developers to access your country package information and computation, you [deploy your own instance](https://github.com/openfisca/openfisca-core#serving-the-api).
```

Comment:
> App Developers? Solely to make this sentence similar to the second one in this page.

(...) you `can` deploy your own instance?

And the link between `deploy your own instance` and the given page might be surprising (goes from deployment to serving). Possible update:
The web API comes with OpenFisca Core module. See the [technical documentation](https://github.com/openfisca/openfisca-core#serving-the-api) for serving instructions.